code/game/objects/effects/aliens.dm

```diff
@@ -162,7 +162,10 @@
 		return
 	if(loc != acid_t.loc && !isturf(acid_t))
 		loc = acid_t.loc
-	ticks += delta_time * acid_strength
+	if(!acid_melt_multiplier)
+		return
+	ticks += delta_time * (acid_strength * acid_melt_multiplier)
+
 	if(ticks >= strength_t)
```

Comment:
early return

Reply:
add

Reply:
!strength_t
Return 

early return???

Reply:
the line the comment is about 4head.
if(ticks >= strength_t)